### ISSUE TITLE
Loading from snapshot: PK-Sim silently creates healthy individuals for unknown disease states (no warning)

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/PK-Sim#3021]
EDIT: The problem is actually this: https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/3021#issuecomment-2511350598

<details><summary>Initial issue description</summary>
Open the snapshot in V11.3 and V12.0 (contains only 1 population simulation **based on the HI population**).
Simulation results look like this (1st pic: PK-Sim **11.3**, 2nd pic: PK-Sim **12.0.390**)
Snapshot and both projects (11.3 and 12.0): [42.zip](https://github.com/user-attachments/files/17977145/42.zip)

<img width="400" src="https://github.com/user-attachments/assets/1e8b8bff-dc17-4650-9713-70ad5c18d892"> <img width="400" src="https://github.com/user-attachments/assets/e54faa49-338c-4c27-b67f-0aaeab663c02">
</details>

**Comment by Yuri05** [comment]
[Midazolam.json.zip](https://github.com/user-attachments/files/17977276/Midazolam.zip) 
[11.3_Midazolam.pksim5.zip](https://github.com/user-attachments/files/17977278/11.3_Midazolam.pksim5.zip) 
[12.390_Midazolam_Full.pksim5.zip](https://github.com/user-attachments/files/17977279/12.390_Midazolam_Full.pksim5.zip)
Actually, already individual simulations greatly differ (both projects again loaded from the snapshot in 11.3 and 12.0)

E.g. compare sim results in the simulation Macgilchrist1986_CPB
<img width="400" src="https://github.com/user-attachments/assets/c28d8a8a-54ef-40d3-b90a-e2723acd53fa"> <img width="400" src="https://github.com/user-attachments/assets/e2cce4e9-463e-40cc-995b-3dbaf10be314">

**Comment by Yuri05** [comment]
Lol, there was no HI population in PK-Sim 11.3, so it just creates **healthy** individuals.
So actually the only bug is that PK-Sim does not produce any warning whatsoever when creating individuals with unknown disease state (and just silently generates them as healthy).
We should at least produce a warning (or maybe **increment the snapshot version** for V12?)
![{70B941A7-620D-49CA-BAE2-049EC8500680}](https://github.com/user-attachments/assets/283a9b02-cf71-458c-a36c-5768b96df4e1)

**Comment by msevestre** [comment]
Yep just wrote the same. I had the feeling this could only work in v12 not in v11.3

**Comment by msevestre** [comment]
>(or maybe increment the snapshot version for V12?)

I don't think so.  Unless you think the snapshot woul dbe loaded in an older version? In that case yes
